frontend/src/CSets.js

```diff
@@ -114,19 +114,24 @@ function CSetsFromDisk(props) {
 const API_ROOT = 'http://127.0.0.1:8000'
 const enclave_url = path => `${API_ROOT}/passthru?path=${path}`
 const backend_url = path => `${API_ROOT}/${path}`
+// let url_concept_sets = enclave_url('objects/OMOPConceptSet')
+let url_concept_sets = backend_url('datasets/csets')
 
 function ConceptSets(props) {
-  let path = 'objects/OMOPConceptSet';
-  let url = enclave_url(path)
+  let url = url_concept_sets
   let navigate = useNavigate();
   const { isLoading, error, data, isFetching } = useQuery([url], () =>
       axios
           .get(url)
-          .then((res) => res.data.data.map(d => d.properties))
-  );
+          .then((res) => res.data.map(d => d.properties))
```

Comment:
The source of the 'map undefined' error. I had a feeling it'd be something like this. If you want, we can add a try/except. I wouldn't be surprised if we get `data.data` in the future. Note that we did get this error both when fetching from (i) enclave directly, and (ii) also from the backend route that reads from disk.